elastalert/kibana_external_url_formatter.py

```diff
@@ -62,6 +70,13 @@ def format(self, relative_url: str) -> str:
         if self.security_tenant:
             long_url = append_security_tenant(long_url, self.security_tenant)
 
+        if self.use_new_shortener:
+            json = { 'locatorId': "elastalert-" + self.id, 'params': { 'url': long_url } }
```

Comment:
I think you want `LEGACY_SHORT_URL_LOCATOR` for the `locatorId`.  See attached har file:

Specifically:

```json
{
          "postData": {
            "mimeType": "application/json",
            "text": "{\"locatorId\":\"LEGACY_SHORT_URL_LOCATOR\",\"params\":{\"url\":\"/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-24h%2Fh,to:now))&_a=(columns:!(),filters:!(),index:%2790943e30-9a47-11e8-b64d-95841ca0b247%27,interval:auto,query:(language:kuery,query:%27%27),sort:!(!(timestamp,desc)))\"}}"
          }
}
```

[kibana.har.zip](https://github.com/jertel/elastalert2/files/7790301/kibana.har.zip)

Reply:
Yeah I'm working on it now and came to the same conclusion.